.drone.yml

```diff
@@ -364,20 +294,8 @@ steps:
     pull: if-not-exists
     image: quay.io/ukhomeofficedigital/kd
     commands:
-      - export TAGGED_VERSION=$${DRONE_COMMIT}
-      - export DOCKER_REGISTRY_URL=$${DOCKER_REGISTRY_URL}
-      - export KUBE_ARTIFACTORY_APP_SECRET=$${PRODUCTION_KUBE_ARTIFACTORY_APP_SECRET}
-      - export KUBE_SERVER=$${PRODUCTION_KUBE_SERVER}
-      - export KUBE_TOKEN=$${PRODUCTION_KUBE_TOKEN}
-      - export PUBLIC_SITE_EXTERNAL_URL=$${PRODUCTION_BASE_URL_SERVICE}
-      - source ./ops/common_lib
-      - source ./ops/prod_values
-      - kd --timeout=6m --file ops/kube/artifactory-secret.yml
-      - kd --timeout=6m --file ops/kube/public-site-network-policy.yml
-      - kd --timeout=6m --file ops/kube/public-site-deployment.yml
```

Comment:
Question: Production only deploys an `external` ingress.
* `deploy.sh` gives both an external and internal deployment.
* Would this be an issue?